src/domain/Ticket.js

```diff
@@ -0,0 +1,31 @@
+import { RULES, TICKET_RULES } from "../util/rule.js";
+
+class Ticket {
+  #numbers;
+  #price;
+
+  constructor({ numbers: numbers }) {
+    this.#setNumbers(numbers, TICKET_RULES.numbersRule, "잘못된 입력입니다");
+    this.#setPrice();
+  }
+
+  get getNumbers() {
+    return this.#numbers;
+  }
```

Comment:
소소한 부분이지만 객체의 불변성을 보장할 수 있도록 습관을 들여보셔도 좋을거 같아요 

```suggestion
  get getNumbers() {
    return [...this.#numbers];
  }
```

Reply:
객체의 불변성을 지켜야, 추적이 쉽고 더 큰 이득이 있다는 것은 알고 있지만 자주 놓치게 되네요 
알려주셔서 감사합니다!
여기서는 객체 불변성을 이용해서 코드 개선이나 테스트 해 볼수 있는 부분이 어떤 것이 있는지 궁금합니다!

Reply:
테스트 코드로 체크를 해본다면 기존 코드에서는 반환 받은 `getNumbers()`를 사용하는 곳에서 임의로 변경한다면 실제 객체 내부에 있는 `#numbers`의 값도 변경되지만, 구조분해할당과 함께 반환한 `getNumbers()`를 사용하는 곳에서 임의로 코드를 변경해도 실제 객체 내부에 있는 `#numbers`의 값은 변경되지 않을거 같아요. 

이런 원리를 이용해서 테스트를 진행해 볼 수 있을거 같네요 !

Reply:
> 테스트 코드로 체크를 해본다면 기존 코드에서는 반환 받은 getNumbers()를 사용하는 곳에서 임의로 변경한다면 실제 객체 내부에 있는 #numbers의 값도 변경되지만, 구조분해할당과 함께 반환한 getNumbers()를 사용하는 곳에서 임의로 코드를 변경해도 실제 객체 내부에 있는 #numbers의 값은 변경되지 않을거 같아요.

약간 이 부분은 리액트의 상태 관리에서도 활용해 볼 수 있을 거 같아요. js에서도 이러한 방식으로 객체를 사용하여, 의도치 않은 클래스 내 인스턴스 변수 변경을 방지할 수 있을 거 같다는 생각이 듭니다! 알려주셔서 감사해요~